test/snippet/io/alignment_file/alignment_file_output.cpp

```diff
@@ -0,0 +1,126 @@
+#include <iostream>
+
+#include <seqan3/alphabet/nucleotide/all.hpp>
+#include <seqan3/io/alignment_file/all.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/range/view/persist.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+
+auto tmp_dir = std::filesystem::temp_directory_path();
+
+// The output snippets will create a /tmp/my.sam
+{
+//! [filename_construction]
+alignment_file_output fout{tmp_dir/"my.sam"}; // SAM format detected, std::ofstream opened for file
+//! [filename_construction]
+}
+
+{
+//! [format_construction]
+// no need to specify the template arguments <...> for format specialization:
+alignment_file_output fout{tmp_dir/"my.sam", fields<field::MAPQ>{}};
+//! [format_construction]
+}
+
+{
+//! [write_range]
+alignment_file_output fout{tmp_dir/"my.sam"};
+
+std::vector<std::tuple<dna5_vector, std::string>> range
+{
+    { "ACGT"_dna5, "First" },
+    { "NATA"_dna5, "2nd" },
+    { "GATA"_dna5, "Third" }
+}; // a range of "records"
+
+fout = range; // will iterate over the records and write them
+//! [write_range]
+}
+
+{
+//! [set_header]
+alignment_file_output fout{tmp_dir/"my.sam"};
+
+// add information to the header of the file.
+fout.header().comments.push_back("This is a comment");
+//! [set_header]
+}
+
+// \todo TODO: uncomment once seqan3::alignment_file_input is implemented.
```

Comment:
This really has the potential to be forgotten...

Reply:
I could create an issue for it...?

Reply:
Or leave it commented out so its visible in the docs? Or add \todo to docs?